src/math/warpExposure.cc

```diff
@@ -535,7 +535,8 @@ int warpImage(DestImageT &destImage, SrcImageT const &srcImage,
     std::vector<double> const srcParentToLocalVec = {static_cast<double>(-srcImage.getX0()),
                                                      static_cast<double>(-srcImage.getY0())};
     auto const srcParentToLocalMap = ast::ShiftMap(srcParentToLocalVec);
-    auto const localDestToSrcMap = srcParentToLocalMap.of(destToSrc.getFrameSet()->of(destLocalToParentMap));
+    auto const localDestToSrcMap =
+            destLocalToParentMap.then(*(destToSrc.getFrameSet())).then(srcParentToLocalMap);
```

Comment:
`Transform`'s constructors are `explicit` because I usually make all constructors explicit under the argument that it's harmless and can avoid trouble (e.g. if a 2 argument ctor is changed to be a 1 argument ctor). But why do you ask?

As to why not `just use destToSrc`: it maps parent coords to parent coords, but C++ indexing of images is done using local coords. The above is needed for the code to work when xy0 is non-zero.